internal/pkg/flink/internal/utils/input.go

```diff
@@ -20,6 +20,11 @@ func GetStdin() *term.State {
 }
 
 func GetConsoleParser() prompt.ConsoleParser {
+	defer func() {
+		if r := recover(); r != nil {
+			log.CliLogger.Warnf("Couldn't open \"/dev/ttv\" file. Error: %v\n", r)
```

Comment:
Any way we can catch this **before** the panic occurs?

Reply:
Oh yeah I guess we can check that this file 1. exists and 2. we have read permission before calling `Open`.